internal/common/voting/election.go

```diff
@@ -86,8 +86,114 @@ func (e *Election) CloseBallot() shared.ClientID {
 }
 
 func (e *Election) bordaCountResult() shared.ClientID {
-	// TODO implement Borda count winner selection method.
-	return e.pluralityResult()
+	// Implement Borda count winner selection method
+	//(may need to modify if methods design is changed)
+	var candidatesNumber int = 0
+	var islandsNumber int
+	var votesLayoutElect map[int][]int
+	votesSliceSquare := e.votes
+	candidatesNumber = len(e.islandsToVote)
+	islandsNumber = len(votesSliceSquare)
+
+	//Initialize votesLayoutMap
+	for i := 1; i < islandsNumber+1; i++ {
+		for j := 0; j < candidatesNumber; j++ {
```

Comment:
instead of iterating over j, just use 
`votesLayoutElect[i] = make([]int, candiatesNumber)`

Reply:
thanks!!